src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,7 +5,18 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int enginePower;
+    private int height;
+
     public Bulldozer() {
+
+    }
+
+    public Bulldozer(int enginePower, int height, String name, String color) {
+        this.enginePower = enginePower;
+        this.height = height;
+        this.setName(name);
+        this.setColor(color);
```

Comment:
we dont need to use `this` keyword here
```suggestion
        setName(name);
        setColor(color);
```